modules/login/templates/form_requestaccount.tpl

```diff
@@ -61,15 +61,19 @@
       <div class="form-group">
           {$form.radiologist.html}
       </div>
+      <div class="form-group">
+        <!-- Google reCaptcha. Don't display if a private key has not bee configured -->
```

Comment:
This is an HTML comment, not a smarty comment (ie. it'll get sent to the client)